UPGRADE-3.x.md

```diff
@@ -160,7 +160,7 @@ ALTER TABLE sn_snippets ADD CONSTRAINT FK_E68115CF30D07CD5 FOREIGN KEY (idUsersC
 #### ArticleBundle
 
 ```sql
```

Comment:
Maybe we could also put those structure changes into a collapsible section to hide if you're using migrations this is irrelevant.

Because there are migrations of changes that are important (like renames etc)

Reply:
no collapsing here in the UPGRADE fro SQL please they are too important to be missed.